packages/terra-form-select/src/_Frame.jsx

```diff
@@ -385,15 +394,21 @@ class Frame extends React.Component {
       customProps.className,
     ]);
 
+    let role;
+
+    if (!disabled) {
+      role = 'combobox';
```

Comment:
For the sake of consistency you could make this a ternary operator just like the rest below.

Reply:
+1 to ternary or creating the aria attributes in a utility method so they are easier to reason about. Something similar to this:

https://github.com/cerner/terra-core/compare/SelectAccessibility#diff-00944ece91b08443a6cd856f48385ee6R10